app/views/articles/index.html.erb

```diff
@@ -1,19 +1,33 @@
 <div class="container">
 
-  <%= react_component("Articles/ArticlesHeader", {title: "Demonic Tails", 
-  subtitle: "Has your cat been possessed?"}) %>
+  <%= link_to root_path do %>
+
+    <%= react_component("Articles/ArticlesHeader", {title: "Demonic Tails", 
+    subtitle: "Has your cat been possessed?"}) %>
+
+  <% end %>
   <br>
 
   <% if @articles %>
```

Comment:
Another possible way to do this is to contain all logic and iteration of articles within React. So you could have a parent React component that will iterate through and render all the articles if present. Could argue that this would make your .erb file a little tidier will less injected ruby code.

Reply:
This is a good suggestion and I'd only add that the less programming logic you have in the `.erb` files the better, as you will be able to add test coverage to them once they are moved into `React` components.

Reply:
I am currently investigating this, I think it would be worth understanding. 

My main concern is the links, that I would no longer be able to use the default rails routing.

Reply:
I have been performing more investigation into how this is done in portal and my initial assumptions do not appear to be correct. Moving this logic to a functional component is an exciting prospect I'll begin to trial.